html-css/week2/styles.css

```diff
@@ -0,0 +1,241 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap');
+
+body {
+  font-family: 'Roboto', sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  text-transform: uppercase;
+  font-weight: 900;
+}
+
+h3 {
+  text-transform: uppercase;
+  font-weight: 900;
+}
```

Comment:
You could combine `h1` and `h3` in one line since both have the same properties. Save the space and write less code